src/components/UI/Card.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { VscEdit } from "react-icons/vsc";
+import { VscSave } from "react-icons/vsc";
+import { VscClose } from "react-icons/vsc";
+
+import "./Card.css";
+
+class Card extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: false,
+      clicked: false,
+      inputValue: "",
+      textareaValue: "",
+      input: "",
+      textarea: ""
+    };
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    this.setState({input: this.state.inputValue});
+    this.setState({textarea: this.state.textareaValue});
```

Comment:
If use combined properties (like 'data' and 'dataValues') - you can use spread operator to copy values. And it won't require any changes if card with be extended with dozen new fields.